client/src/components/Button/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import {
+  faArrowLeft,
+  faArrowRight,
+  faCheckCircle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import './style.css'
+
+export default function Button(props) {
+  const { label, style, done, rightIcon, leftIcon, onClick } = props;
```

Comment:
as you named rightIcon and leftIcon make the done => doneIcon

Reply:
Done